t/app/helpers/send_email.t

```diff
@@ -33,16 +34,74 @@ my @emails = Email::Send::Test->emails;
 is scalar(@emails), 1, "caught one email";
 
 # Get the email, check it has a date and then strip it out
-my $email_as_string = $emails[0]->as_string;
-ok $email_as_string =~ s{\s+Date:\s+\S.*?$}{}xms, "Found and stripped out date";
-ok $email_as_string =~ s{\s+Message-ID:\s+\S.*?$}{}xms, "Found and stripped out message ID (contains epoch)";
+my $email_as_string = get_first_email(@emails);
 
-my $expected_email_content =   file(__FILE__)->dir->file('send_email_sample.txt')->slurp;
+my $expected_email_content =   path(__FILE__)->parent->child('send_email_sample.txt')->slurp;
 my $name = FixMyStreet->config('CONTACT_NAME');
 $name = "\"$name\"" if $name =~ / /;
 my $sender = $name . ' <' . FixMyStreet->config('DO_NOT_REPLY_EMAIL') . '>';
 $expected_email_content =~ s{CONTACT_EMAIL}{$sender};
 
-is $email_as_string,
-$expected_email_content,
-  "email is as expected";
+is_string $email_as_string, $expected_email_content, "email is as expected";
+
+subtest 'MIME attachments' => sub {
+    my $data = path(__FILE__)->parent->child('grey.gif')->slurp_raw;
+
+    Email::Send::Test->clear;
+    my @emails = Email::Send::Test->emails;
+    is scalar(@emails), 0, "reset";
+
+    ok $c->send_email( 'test.txt',
+        { to => 'test@recipient.com',
+          attachments => [
+             {
+                body => $data,
+                attributes => {
+                    filename => 'foo.gif',
+                    content_type => 'image/gif',
+                    encoding => 'quoted-printable',
```

Comment:
I wonder if it's worth changing this to `base64` so it matches what `munge_sendreport_params` actually uses.